### PR TITLE
Golang style

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,8 +7,8 @@ install-tools:
 
 VERIFY += generate
 .PHONY: generate
-generate: CODE_GEN_DIR = pkg/json
-generate: GEN = pkg/cmd/codegenerator/generator.go
+generate: CODE_GEN_DIR = pkg/json/schemas
```
Comment (roytman) [inline Makefile:10]: the generated types are not schemas, they are possible types used by the system

**Review by roytman — COMMENTED** [review]
I am not sure about renaming to `schema`

File: Makefile [hunk 2]
```diff
@@ -7,8 +7,8 @@ install-tools:
...
+generate: GEN = pkg/cmd/generator/generator.go
 generate:
 	go run $(GEN) -s ./json/ovn-nb.ovsschema -d $(CODE_GEN_DIR)
```
Comment (roytman) [inline Makefile:13]: on the second thought,  maybe we should rename ./json dir to ./schemas

Comment (roytman) [reply Makefile:13]: I meant at the top json directory - go run $(GEN) -s ./json/ovn-sb.ovsschema -d $(CODE_GEN_DIR) ->
go run $(GEN) -s ./schemas/ovn-sb.ovsschema -d $(CODE_GEN_DIR)

**Comment by roytman** [comment]
please rebase

File: Makefile (hunk 1 above)
Comment (hunchback) [reply Makefile]: moved to pkg/types

File: Makefile (hunk 2 above)
Comment (hunchback) [reply Makefile:13]: no sure ...

Comment (roytman) [reply Makefile:13]: I'm talking about the input directory, where we store schemas(the `-s` flag of the  generator)

Comment (hunchback) [reply Makefile:13]: ok moved form `json` to `schemas`

**Pull request merged**